### PR TITLE
Add documentation and class descriptions

### LINKED ISSUE #62: Add documentation for `SolubilityRatioMaterial`
## Reason
`SolubilityRatioMaterial` is not documented.

## Design
An .md file should be added for `SolubilityRatioMaterial`.

## Impact
Improved documentation.

### PR DESCRIPTION
**Pull request by cticenhour**
Application-level reports now pass and will be changed to an error for future PRs. 

Closes #62

### CODE REVIEW

File: doc/content/source/materials/SolubilityRatioMaterial.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# SolubilityRatioMaterial
+
+!syntax description /Materials/SolubilityRatioMaterial
+
+## Overview
+
+The SolubilityRatioMaterial object is used to calculate the jump in specie concentration across a
+material interface, given the solubility values for those materials and specie concentration on
+either side of the interface. Solubilities for the "primary" and "secondary" sides can be provided
```
Comment (simopier) [inline doc/content/source/materials/SolubilityRatioMaterial.md:9]:
```suggestion
The SolubilityRatioMaterial object is used to calculate the jump in species concentration across a
material interface, given different solubility values for those materials and species concentration on
either side of the interface. Solubilities for the "primary" and "secondary" sides can be provided
```

File: doc/content/source/materials/SolubilityRatioMaterial.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+via the [syntax/Materials/index.md] using automatic differentiation (a.k.a. an
+`ADMaterialProperty`).
+
+The solubility ratio jump is calculated using the following relationship:
+
+!equation
+J = \frac{c_p}{S_p} - \frac{c_s}{S_s}
+
+where $J$ is the calculated solubility ratio jump (available as an `InterfaceMaterial` property,
+named the `solubility_ratio`), $c_i$ is the specie concentration on the $i$th side of the
```
Comment (simopier) [inline doc/content/source/materials/SolubilityRatioMaterial.md:19]:
```suggestion
named the `solubility_ratio`), $c_i$ is the species concentration on the $i$th side of the
```

**Review by simopier — COMMENTED** [review]
Thank you for that!

I think `species` has an s even in singular form, doesn't it?

**Comment by cticenhour** [comment]
I've seen it as "specie", but that's not saying those folks were correct. 😄 I'll accept these for consistency across all our documentation.

**Review by simopier — APPROVED** [review]

**Pull request merged**